content/en/blog/2023/end-user-discussions-02.md

```diff
@@ -0,0 +1,229 @@
+---
+title: OpenTelemetry End-User Discussions Summary for February 2023
+linkTitle: End-User Discussions Feb 2023
+date: 2023-02-27
+author: >-
+  [Pranay Prateek](https://github.com/pranay01) (SigNoz)
+---
+
+_With contributions from [Henrik Rexed](https://github.com/henrikrexed)
+(Dynatrace), [Michael Hausenblas](https://github.com/mhausenblas) (AWS),
+[Rynn Mancuso](https://github.com/musingvirtual) (Honeycomb),
+[Reese Lee](https://github.com/reese-lee) (New Relic) and
+[Adriana Villela](https://github.com/avillela) (Lightstep)_
+
+The OpenTelemetry end-user group meet takes place every month for users in the
+Americas (AMER), Europe Middle-East & Africa (EMEA), and Asia-Pacific (APAC).
+
+The discussions take place using a
+[Lean Coffee format](https://agilecoffee.com/leancoffee/), whereby folks are
+invited to post their topics to the
+[Agile Coffee board like this one](http://agile.coffee/#3716060f-183a-4966-8da4-60daab2842c4),
+and everyone in attendance votes on what they want to talk about.
+
+## What we talked about
+
+Some interesting topics that were discussed this month were:
+
+- Sampling for traces
+- Emitting business metrics
+- Monitoring the health of OpenTelemetry Collector
+- Backup/Buffer capabilities of OTel Collector
+
+## Discussion Highlights
+
+Below is the summary of this month's discussions.
+
+### OpenTelemetry Collector
+
+#### 1 - Monitoring OTel Collector's health
+
+**Q:** Are there any suggestions for monitoring OTel Collector's health or
+patterns for collecting agent telemetry?
+
+**A:** Collectors can be used to collect telemetry data from other Collectors,
+which doesn't really need to be a disparate telemetry system. Users should also
+think about collecting multiple signals so that even if one signal fails, they
+get alerted by another. Here's an
+[article](https://ref.otel.help/otel-collector-ops/) discussing this.
+
+#### 2 - Timeline for opamp extension
+
+**Q:** Is there any timeline for implementing the
+[opamp spec](https://github.com/open-telemetry/opamp-spec) for agent management?
+
+**A:** It's not a top priority as of now. It would be good to have a maintainer
+from the community for opamp. You can track the issue
+[here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16462).
```

Comment:
For future reference, avoid having links with link text "here". (I'll fix one, but leave the rest for now.)

```suggestion
from the community for OpAMP. To track progress, see 
[issue #16462](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16462).
```